testsuite/tools/expect_test.ml

```diff
@@ -0,0 +1,402 @@
+(***********************************************************************)
+(*                                                                     *)
+(*                                OCaml                                *)
+(*                                                                     *)
+(*                 Jeremie Dimino, Jane Street Europe                  *)
+(*                                                                     *)
+(*  Copyright 2016 Jane Street Group LLC                               *)
+(*                                                                     *)
+(*  All rights reserved.  This file is distributed under the terms of  *)
+(*  the GNU Lesser General Public License version 2.1, with the        *)
+(*  special exception on linking described in the file ../LICENSE.     *)
```

Comment:
There is a new rule for copyright headers: they must simply mention `LICENSE` without a path.